packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5672,7 +5686,10 @@ export default class Meeting extends StatelessWebexPlugin {
       // @ts-ignore
       this.webex.internal.newMetrics.submitClientEvent({
         name: 'client.media.tx.start',
-        payload: {mediaType: data.type},
+        payload: {
+          mediaType: data.type,
+          shareInstanceId: data.type === 'share' && this.shareInstanceId,
```

Comment:
nitpick: this will put `shareInstanceId: false` in the event for types other than "share" (so for main audio and video). Are we sure that Call Analyzer will be OK with it? I guess they just ignore it, but maybe it would be better to set it to undefined for those cases?